### PR TITLE
Code required to interact with Anti-Brownout in core 751

### PR DESCRIPTION
**Pull request by Kevin-Woodman**
The Anti-Brownout code (should) work with out this but just not adjustable.

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -31,6 +32,12 @@ public void robotInit() {
     // Instantiate our RobotContainer.  This will perform all our button bindings, and put our
     // autonomous chooser on the dashboard.
     m_robotContainer = new RobotContainer();
+
+    SmartDashboard.putNumber("Speed Divider", 11);
+    SmartDashboard.putNumber("Speed Cap Max", 0.9);
+  
+    SmartDashboard.putNumber("Speed Cap", 0.9);
+    SmartDashboard.putBoolean("Anit-Brownout mode", true);
```
Comment (InsightGit) [inline src/main/java/frc/robot/Robot.java:40]: i love me some anit-brownout mode

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -21,9 +22,18 @@
...
+    public static int kI;
+    public static int kD;
+
     public static int LEDPort = 0;
     public static int LEDLength = 10;
 
     public static Port colorSensorPort = Port.kOnboard;
     public static float proximityThreshhold = 200;
+
+    public static int driveStickPort = 0;
```
Comment (InsightGit) [inline src/main/java/frc/robot/Constants.java:31]: These are good for now with testing with 2019, but we will eventually need to change them to adapt to the new drivetrain with neos.

**Review by InsightGit — CHANGES_REQUESTED** [review]
Looks good. I will handling merging(and switch this PR to approved) when the corresponding core751 PR gets approved.

**Review by dsaliba — CHANGES_REQUESTED** [review]
Make it compile and ideally make this system a bit more robust, I would like to see a brownout handler we can construct that will allocate % of current to different components provide them with their modifiers accordingly, this may involve rebasing so you have more subsystems to work with. I can talk to you about this and i would also be willing to write at least a skeleton for it if you want. Other than that great work the drop in brownouts is very impressive.

**Comment by InsightGit** [comment]
@dsaliba the reason it doesn't compile is because the upstream core751 PR #2 hasn't been merged yet.

**Pull request closed without merging**